ansible/examples/slave_libvirt_static.yml

```diff
@@ -0,0 +1,223 @@
+      lineinfile:
+        dest: "/home/{{ jenkins_user }}/.bashrc"
+        regexp: '^export LC_ALL='
+        line: "export LC_ALL=en_US.UTF-8"
+        create: true
+        state: present
+
+    - name: set utf-8 for LANG
+      lineinfile:
+        dest: "/home/{{ jenkins_user }}/.bashrc"
+        regexp: '^export LANG='
+        line: "export LANG=en_US.UTF-8"
+
+    - name: set utf-8 for LANGUAGE
+      lineinfile:
+        dest: "/home/{{ jenkins_user }}/.bashrc"
+        regexp: '^export LANGUAGE='
+        line: "export LANGUAGE=en_US.UTF-8"
+
+    - name: ensure the build dir exists
+      file:
+        path: "/home/{{ jenkins_user }}/build"
+        state: directory
+        owner: "{{ jenkins_user }}"
+
+    - name: ensure the home dir has the right owner permissions
+      file:
+        path: "/home/{{ jenkins_user }}"
+        state: directory
+        owner: "{{ jenkins_user }}"
+        group: "{{ jenkins_user }}"
+        recurse: yes
+
+    # Makes sure only our block is in the file
+    - name: remove gitconfig file
+      file:
+        path: "/home/{{ jenkins_user }}/.gitconfig"
+        state: absent
+
+    - name: ensure our gitconfig file exists
+      blockinfile:
+        dest: "/home/{{ jenkins_user }}/.gitconfig"
+        create: yes
+        owner: "{{ jenkins_user }}"
+        group: "{{ jenkins_user }}"
+        block: |
+          [user]
+          name=Ceph CI
+          email=ceph-release-team@redhat.com
+
+    - name: Set Hostname with hostname command
+      hostname:
+        name: "{{ ansible_hostname }}"
+
+    - name: ensure that the current host is in /etc/hosts. Yes this is a thing.
+      replace:
+        backup: yes
+        dest: /etc/hosts
+        regexp: '^(127\.0\.1\.1(?!.*\b{{ ansible_hostname }}\b).*)$'
+        replace: '\1 {{ ansible_hostname }}'
+
+    - name: install six, latest one
+      pip:
+        name: six
+        state: latest
+
+    - name: install python-jenkins
+      # HORRIBLY BROKEN. This is temporary until this lands upstream:
+      # https://github.com/ceph/python-jenkins/commit/8e018bf7d88dfc308833d195a6ebd29231a8969d
+      # https://review.openstack.org/460363
+      # Still not in upstream pip version as of 22JUN2017
+      pip:
+        name: git+https://github.com/ceph/python-jenkins@patched#egg=python-jenkins
+
+    - name: add github.com host key
+      known_hosts:
+        path: '/etc/ssh/ssh_known_hosts'
+        # we need to use 'host' here because prado currently uses ansible-playbook==1.9.1
+        host: 'github.com'
+        # github.com.pub is the output of `ssh-keyscan github.com`
+        key: "{{ lookup('file', 'files/ssh/hostkeys/github.com.pub') }}"
+
+    - name: start the libvirt-bin service
+      service:
+        name: libvirt-bin
+        state: started
+
+    - name: start the libvirt-guests service
+      service:
+        name: libvirt-guests
+        state: started
+
+    - name: register the new slave to jenkins master with jnlp
```

Comment:
Right.  Since the playbook is only good for static slaves and the only static slaves we have at the moment are in Sepia, jnlp is the only way to connect.